examples/pytorch/test_accelerate_examples.py

```diff
@@ -321,3 +322,24 @@ def test_run_semantic_segmentation_no_trainer(self):
             run_semantic_segmentation_no_trainer.main()
             result = get_results(tmp_dir)
             self.assertGreaterEqual(result["eval_overall_accuracy"], 0.10)
+
+    def test_image_classification_no_trainer(self):
+        tmp_dir = self.get_auto_remove_tmp_dir()
+        testargs = f"""
+            run_image_classification_no_trainer.py
+            --dataset_name huggingface/image-classification-test-sample
+            --output_dir {tmp_dir}
+            --num_warmup_steps=8
+            --learning_rate=3e-3
+            --per_device_train_batch_size=2
+            --per_device_eval_batch_size=1
+            --checkpointing_steps epoch
```

Comment:
```suggestion
            --checkpointing_steps epoch
            --seed 42
```
This helps with making the test locally pass for me.